Update FAQ to reflect current changes in Open API (#66)
SAP/cloud-sdk: docs/java/faq.mdx

Some of them even have known flaws for which we have workarounds.
If you can't solve your issue via debugging and experimenting, please, report your incident via [Stack Overflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk) or our GitHub repositories.

## Open API Related Questions

> ### Do You Support Open API Client Libraries?

Author: ```suggestion
### Do You Support RESTful Services?
```

> Yes, we do for Open API 3.0 specifications.

Author: ```suggestion
Yes, we do support [OpenAPI 2.0 also known as Swagger](https://swagger.io/specification/v2/), and  [OpenAPI 3.0](https://swagger.io/specification/) specifications. The latter is preferred.
```

> Yes, we do support [OpenAPI 2.0 also known as Swagger](https://swagger.io/specification/v2/), and  [OpenAPI 3.0](https://swagger.io/specification/) specifications. The latter is preferred.

Author: ```suggestion
Yes, we do support [OpenAPI 2.0 also known as Swagger](https://swagger.io/specification/v2/), and  [OpenAPI 3.0](https://swagger.io/specification/) specifications.
The latter is preferred.
```

Find documentation how to generate a [type-safe Open API client here](features/rest/generate-rest-client.mdx).

> We also release pre-generated libraries in colaboration with service providers who release restful Open API services.

Author: ```suggestion
We also release pre-generated libraries in collaboration with service providers who release RESTful Open API services.
```
